package/harvester-os/files/usr/sbin/migration/migrate_harv_mgmt_to_mgmt_br.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+HARV_CONFIG="${HARV_CONFIG:-/oem/99_custom.yaml}"
+HARV_MGMT="harvester-mgmt"
+IFCFG=
+IFROUTE=
+
+function migrate_mgmt_config () {
+	MODE=
+	# search all files
+	for (( i=0; ; i++ ))
+	do
+		if ! yq -e ".stages.initramfs[0].files[$i]" "$HARV_CONFIG" > /dev/null 2>&1; then
+			echo "end files"
+			break
+		fi
+
+		if yq ".stages.initramfs[0].files[$i].path" "$HARV_CONFIG" | grep -q "/etc/sysconfig/network/ifcfg-${HARV_MGMT}"; then
```

Comment:
Our /oem/99_custom.yaml will be generated from the installer result.
In installer config.yaml, yaml object "key" will be the interface name and file name also.
If we have several interface in installer config.yaml, it will not have the same entry.
So `/etc/sysconfig/network/ifcfg-${HARV_MGMT}` will be one and only.

If users add some files in the /oem/99_custom.yaml with the same filename, we could only use the first file to migrate the config.
And in our guide, we always ask users to modify the original files rather than adding a new file to replace